ext/node/polyfills/internal/crypto/keys.ts

```diff
@@ -215,6 +215,12 @@ export interface JsonWebKeyInput {
 export function prepareAsymmetricKey(key) {
   if (isStringOrBuffer(key)) {
     return { format: "pem", data: getArrayBufferOrView(key, "key") };
+  } else if (isKeyObject(key)) {
```

Comment:
interesting that this works in node but docs don't mention about KeyObject being a valid input.